schemas/tlc/1.3.0/sxl.yaml

```diff
@@ -27,11 +27,10 @@ objects:
           fault
       6:
         title: Connected / Normal - In Use
+        description: Traffic Light Controller is functioning normally
       7:
         title: Connected / Normal - Idle
-        description: |-
-          Traffic Light Controller dark according to configuration.
-          NOTE! When dark according to configuration the controller is considered to be in use
+        description: Traffic Light Controller is in dark mode or yellow flash due to fault
```

Comment:
IF 7 is Normal mode (although idle) why is fault mentioned?

Reply:
The original issue was opened 5 years ago and I can't remember the reasoning behind this any longer. I suggest we simply use this bit for dark mode.